File: app/src/main/java/ua/com/amicablesoft/android/wr/app/WorkoutReporterApplication.java
package ua.com.amicablesoft.android.wr.app;

import android.app.Application;

import com.crashlytics.android.Crashlytics;

import io.fabric.sdk.android.Fabric;

/**
 * Created by lapa on 21.10.16.
 */

public class WorkoutReporterApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        Fabric.with(this, new Crashlytics());
    }
}

File: app/src/main/java/ua/com/amicablesoft/android/wr/app/WorkoutReporterApplication.kt
/*
 * Created by Olha Tymoshenko on 5/24/18 8:34 PM.
 * Copyright (c) 2018. All rights reserved.
 */

package ua.com.amicablesoft.android.wr.app

import android.app.Application
import com.crashlytics.android.Crashlytics
import io.fabric.sdk.android.Fabric

class WorkoutReporterApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        Fabric.with(this, Crashlytics())
    }
}